### ISSUE TITLE
Feature request : deploy specific branch

### ISSUE THREAD

**Issue by cesarvarela** [Azure/azure-xplat-cli#3264]
Hi, not sure if this should be here, but anyway:

I would like to be able to deploy a specific branch when using git local deployment, something similar to what heroku does:

```
 git push heroku yourbranch:master
```

For Node.JS apps at least.

**Comment by amarzavery** [comment]
@cesarvarela - I assume you are talking w.r.t webapps/websites created in Azure, correct ?
@LukaszStem, @ahmedelnably  @naveedaz - Can you provide a timeline for this feature request ?

**Comment by cesarvarela** [comment]
yes!

**Comment by naveedaz** [comment]
I dont think we are taking any new feature requests for azure-xplat-cli anymore. @LukaszStem Is there some way of achieving this currently?

**Comment by cesarvarela** [comment]
@naveedaz What do you mean by that? Is this deprecated?
